services/api/src/resources/billing/resolvers.ts

```diff
@@ -0,0 +1,136 @@
+import * as R from 'ramda';
+import billingModel, { AddBillingModifierInput, BillingModifier, BillingModifiersInput, UpdateBillingModifierInput, DeleteBillingModifierInput, DeleteAllBillingGroupModifiersInput } from '../../models/billing';
```

Comment:
line is too long, I only bring it up to ask if you have formatting setup in vscode

Reply:
prettier was giving me a nightmare formatting things wrong so I turned it off. I'll try and get that setup again.